controlPanel/controlpanel.go

```diff
@@ -112,15 +112,15 @@ func New(config *Config) {
 	controlPanel.MsgInputSubscription.Subscribe(pubsub.GetPath(config.NodeName, "bmv", "rest"))
 
 	// internal events
-	controlPanel.MovedToHeightSubscription.Subscribe(pubsub.GetPath(config.NodeName, event.Path.Seq))
-	controlPanel.BalanceChangedSubscription.Subscribe(pubsub.GetPath(config.NodeName, event.Path.Bank))
-	controlPanel.DBlockCreatedSubscription.Subscribe(pubsub.GetPath(config.NodeName, event.Path.Directory))
+	controlPanel.MovedToHeightSubscription.Subscribe(pubsub.GetPath(config.NodeName, events.Path.Seq))
```

Comment:
I renamed the event package to events. When working with instances of an event, "event" is a more suitable name, and that collided with the package name.